packages/sdk/src/engine-commands/queryEngineCommons.ts

```diff
@@ -77,7 +77,7 @@ export function loadNodeAPILibrary(queryEnginePath: string) {
       () => load<NodeAPILibraryTypes.Library>(queryEnginePath),
       (e) => ({
         type: 'connection-error' as const,
-        reason: 'Unable to establish a connection to query-engine-node-api library',
+        reason: `Unable to establish a connection to query-engine-node-api library. Have you tried installing 'openssl'?`,
```

Comment:
I don't think this is usually caused by missing OpenSSL - that is just one of many possible reasons.
(If we want to react to OpenSSL missing explicitly, we should look at the detailed error message only and trigger by that and append something to the error string)